README.md

```diff
@@ -1,32 +1,51 @@
 Overview
 ========
 
-Token SDK for iOS. The SDK is built on top of protobuf gRPC API definition.
+Token SDK for iOS. Docs https://developer.token.io/sdk/
 
-Pick up new protos
-------------------
+The SDK is built on top of protobuf gRPC API definition,
+https://developer.token.io/sdk/pbdoc/io_token_proto_gateway.html
+
+Client Usage
+------------
+
+To use the SDK version compatible with Token's Sandbox testing
+environment (typical usage), use the published pod:
+
+```
+  pod 'TokenSdk', '~> 1.0.72'
```

Comment:
pod 'TokenSdk' only
the 1.0.72 setting means the project needs TokenSdk version later then 1.0.72, which will not be used for most external users.

Reply:
or if this is a breaking version that we suggest people avoid the older version

Reply:
another thing is we need 

source 'https://github.com/tokenio/token-cocoa-pods.git'

Since our spec is not in the official public spec storage. 
We need to make this git repo public too.

Reply:
> source 'https://github.com/tokenio/token-cocoa-pods.git'

D'oh!